### PR TITLE
Add blog on advanced vim

### PR DESCRIPTION
**Pull request by iamshubhankarkhare**
#### Please note that in case of below checklist not updated accordingly, the maintainers have the right to immediately close your pull request.

- [x] You created a branch from `dev` branch.
- [x] You PR is raised on `dev` branch and not on`master`.
- [x] Blog follows [Blog Guidelines](https://github.com/LoginRadius/engineering-portal/blob/master/GUIDELINES.md).
- [x] You have read the [Contribution Guidlines](https://github.com/LoginRadius/engineering-portal/blob/master/CONTRIBUTING.md) before creating this PR.
- [x] Checked for any duplicate blog entry.
- [x] Checked for all grammatical errors in the blog.
- [x] You have not published the blog content anywhere else on the internet on platforms like medium, dev.to etc.
- [x] The content is not copy-pasted from anywhere else on the Internet.

##### Description of change
Added a new blog on Vim

##### Author Details for Blog Post Entry
  id: Shubhankar Khare
  bio: A diligent and prompt learner, Shubhankar Khare is a passionate full stack web developer from India. He is crazy about 
  learning new technologies and is an open source enthusiast.
  github: "iamshubhankarkhare"
  linkedin: "shubhankar-khare"

### CODE REVIEW

**Comment by iamshubhankarkhare** [comment]
@vyasriday @puneetsingh24 please review..

File: content/vim-level-up/index.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+To create a new buffer in the same view -- creating a split in the current window -- use `:sp filename`. 
+
+For a vertical split, use `:vsp filename`.
+
+> To be honest, using buffer numbers to switch buffers is not what I prefer to do. I prefer to switch based on what I see, like `ctrl-w` then `j`to focus on a buffer below if I'm splitting the window. `:tabprevious` to move a tab on the left if I'm using tabs.
+
+Here are some key bindings you might want to use for a smooth workflow.
+
+```
+nnoremap <C-t> :tabnew<Cr>
```
Comment (Akash-Patil) [inline content/vim-level-up/index.md:107]: Change `<Cr>` to `<CR>` for "carriage return" as done in other cases.

File: content/vim-level-up/index.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+Now, to repeat your macro 99 times:
+
+```99@1```
+
+And that's all.
+
+![macro demo](vim-macro-demo.png)
+
+## Buffers & Splitting
+In simple terms, a _buffer_ is a file that has been loaded into memory. Everything you do in VIM is in a buffer. You can list all the buffers by `:Buffers`. And then, if you want to check out a particular buffer, use `:b n` -- where n is the buffer number.
```
Comment (Akash-Patil) [inline content/vim-level-up/index.md:94]: Should be `:buffers` instead of `:Buffers`. Please confirm this.

**Review by Akash-Patil — CHANGES_REQUESTED** [review]
Hi @iamshubhankarkhare,

I have reviewed the document and suggested a few minor changes. Please have a look. 

Regards,
@Akash-Patil

**Review by mohammed786 — APPROVED** [review]

**Pull request merged**